service/src/main/kotlin/fi/espoo/evaka/reservations/AttendanceReservationController.kt

```diff
@@ -351,6 +353,245 @@ class AttendanceReservationController(
+                                                DailyServiceTimesValue.VariableTimes(
+                                                    validityPeriod =
+                                                        basicInfoItem.validityPeriod
+                                                            ?: DateRange(
+                                                                examinationDate,
+                                                                examinationDate
+                                                            )
+                                                )
+                                            else -> null
+                                        }
+                                )
+                            }
+
+                    DailyChildReservationResult(
+                        children = childMap,
+                        childReservations = childReservationInfos
+                    )
+                }
+            }
+            .also {
+                Audit.ChildReservationStatusRead.log(
+                    targetId = unitId,
+                    meta = mapOf("childCount" to it.children.size)
+                )
+            }
+    }
+
+    data class GroupReservationStatisticResult(
+        val calculatedPresent: BigDecimal,
+        val presentCount: Int,
+        val absentCount: Int,
+        val groupId: GroupId?
+    )
+
+    data class DayReservationStatisticsResult(
+        val date: LocalDate,
+        val groupStatistics: List<GroupReservationStatisticResult>
+    )
+
+    @GetMapping("/confirmed-days/stats")
+    fun getReservationStatisticsForConfirmedDays(
+        db: Database,
+        user: AuthenticatedUser,
+        clock: EvakaClock,
+        @RequestParam unitId: DaycareId
+    ): List<DayReservationStatisticsResult> {
+        return db.connect { dbc ->
+                dbc.read { tx ->
+                    ac.requirePermissionFor(
+                        tx,
+                        user,
+                        clock,
+                        Action.Unit.READ_UNIT_RESERVATION_STATISTICS,
+                        unitId
+                    )
+
+                    val unitData =
+                        tx.getDaycare(unitId) ?: throw BadRequest("Invalid unit id $unitId")
+                    val nonReservableRange =
+                        FiniteDateRange(
+                            clock.today().plusDays(1),
+                            getNextReservableMonday(
+                                    clock.now(),
+                                    featureConfig.citizenReservationThresholdHours,
+                                )
+                                .minusDays(1)
+                        )
+
+                    val nextConfirmedUnitDays =
+                        nonReservableRange
+                            .dates()
+                            .filter { unitData.operationDays.contains(it.dayOfWeek.value) }
```

Comment:
This is probably where holidays should be taken into account.

Reply:
Holidays removed from date selection using `getUnitOperationalDayData` from desktop calendar code. As I understand holiday periods should not affect the day selection?